apps/plotly_examples/src/data/parsed_data.json

```diff
@@ -0,0 +1,4331 @@
+[
+  {
+    "data": [
+      {
+        "type": "scatter",
```

Comment:
This file is very big. Could you move 1 case to 1 file for better readability and maintenance.

Reply:
so there will be 100 such files?

Reply:
yes

Reply:
You can take the comment as follow up.